### PR TITLE
[FIX] l10n_nl: set correct VAT codes on fiscal position lines

### PR DESCRIPTION
**Pull request by Yenthe666**
Description of the issue/feature this PR addresses: Set the correct taxes to fiscal position lines

Current behavior before PR:
Before this PR quite a lot of the lines on the fiscal positions "EU landen" and "Niet-EU landen" where mapped wrong, some of the lines even referred to another tax percentage. For example:
The tax "BTW te vorderen laag (inkopen) 9%" on the product mapped to the tax to apply named "Inkopen import buiten EU laag 6%", which should be "Inkopen import buiten EU laag 9%" to correctly counterpart the taxes. All the other changes where similar, either invalid tax rates where applied or it mapped to a wrong category. For example:
"Verkopen/omzet overig diensten" on the product would map to "Installatie/afstandsverkopen buiten EU" as the VAT to apply. This should map to "Verkopen export buiten EU".

Desired behavior after PR is merged:
All the fiscal position lines map to the corresponding taxes.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Yenthe666** [comment]
Hi @jco-odoo,

This is the second PR to fix the Dutch accounting (in V12). It will fix 20 fiscal position lines that mapped to an invalid VAT. Could you please check this one & merge it in? We believe we've got them all correct now and I did quite some testing.
CC @ploegvde 

Regards,
Yenthe

**Comment by Yenthe666** [comment]
@jco-odoo sorry to bother again but could we get this one checked soon? It is quite annoying when this gets installed and you have to manually remap all of them.

**Comment by jco-odoo** [comment]
The diensten vs producten is something you require for the tax report?  It seems like a thing of the Netherlands.  The same thing as for diensten was done with producten?  I would like to have a quick try myself before merging.

**Comment by Yenthe666** [comment]
Hi,

I don't really get your question, could you rephrase it or explain in more detail? Right now the fiscal position lines map to the wrong taxes which would result in results on wrong parts of the official tax report yes.
Let me know if you need more input or details. :)

**Comment by jco-odoo** [comment]
I understand what you want to do with the corrections.  Just wanted to understand the context better.

**Comment by Yenthe666** [comment]
Awesome. Just let me know if there is anything I can do to help and thanks for the quick feedback.

File: addons/l10n_nl/data/account_fiscal_position_tax_template.xml [hunk 1]
```diff
@@ -34,7 +34,7 @@
         <record id="position_tax_extracom_d_0" model="account.fiscal.position.tax.template">
             <field name="position_id" ref="fiscal_position_template_non_eu"/>
             <field name="tax_src_id" ref="btw_0_d"/>
-            <field name="tax_dest_id" ref="btw_X3"/>
+            <field name="tax_dest_id" ref="btw_X1"/>
```
[the remark this answers is not in the document]
Comment (Yenthe666) [reply addons/l10n_nl/data/account_fiscal_position_tax_template.xml]: @astirpe nope, I disagree.
`btw_X3` maps to "Installatie/afstandsverkopen buiten EU" while `btw_X1` maps to "Verkopen export buiten EU". See https://github.com/odoo/odoo/blob/c7a281d88004fee2c20f9000825b4341fe951fac/addons/l10n_nl/data/account_tax_template.xml#L855-L867
Why would you think it should map to "Installatie/Afstandsverkopen buiten EU"? Do you have something that proves your statement?

[the remark this answers is not in the document]
Comment (astirpe) [reply addons/l10n_nl/data/account_fiscal_position_tax_template.xml]: Was my opinion, not a statement.

[the remark this answers is not in the document]
Comment (Yenthe666) [reply addons/l10n_nl/data/account_fiscal_position_tax_template.xml]: Ahh I see, do you have an idea/reason why you'd like it the other way around though? I'm curious.

[the remark this answers is not in the document]
Comment (astirpe) [reply addons/l10n_nl/data/account_fiscal_position_tax_template.xml]: I don't see a valid reason for making those changes (btw_X3 -> btw_X1). Is there something that can prove that those changes are required?

**Comment by Yenthe666** [comment]
@jco-odoo did you get around to checking this?

File: addons/l10n_nl/data/account_fiscal_position_tax_template.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (astirpe) [reply addons/l10n_nl/data/account_fiscal_position_tax_template.xml]: So at the end no explanations about the change you are doing? Just marked as resolved?

**Comment by ploegvde** [comment]
@Yenthe666 @astirpe "Installatie/afstandsverkopen buiten EU" is used when you deliver products or services to consumers inside the EU. These consumers don't have a TIN. So this is a B2C delivery within the EU and that is only allowed until a certain amount (See: https://www.belastingdienst.nl/wps/wcm/connect/bldcontentnl/belastingdienst/zakelijk/btw/zakendoen_met_het_buitenland/goederen_en_diensten_naar_andere_eu_landen/export_van_specifieke_goederen_en_in_bijzondere_situaties/export_naar_klanten_die_geen_btw_aangifte_doen_drempelbedragen/drempelbedragen_afstandsverkopen).

If you go above that amount you have to calculate the TAX of the country where you deliver and you need to do tax reporting in that country. So that needs extra country taxes and needs to be solved separately.

So by default you calculate for "Installatie/afstandsverkopen binnen EU" 21% VAT. That is now not set and wrong.

So the change is correct and logic, because most deliveries are B2B. For B2C indeed we need to use the "Installatie/afstandsverkopen binnen EU", but i think we can better make a new Fiscal Position with the same name for that.

So 2 changes needed:
1. Installatie/afstandsverkopen binnen EU >> Set 21% on that tax
2. New Fiscal position "Installatie/afstandsverkopen binnen EU" where we map the default taxes to the "Installatie/afstandsverkopen binnen EU" tax.

File: addons/l10n_nl/data/account_fiscal_position_tax_template.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (astirpe) [reply addons/l10n_nl/data/account_fiscal_position_tax_template.xml]: Ok thanks.
Why the two changes asked by @ploegvde cannot be included within this PR?

**Comment by Yenthe666** [comment]
@jco-odoo can I ask you for an update please?

**Comment by jco-odoo** [comment]
@Yenthe666 You did not do the changes requested by @ploegvde

**Comment by Yenthe666** [comment]
@jco-odoo so if I made a second fiscal position (in this PR) with the extra configurations you'll merge this one in? You'd like it in one PR?
